packages/airnode-protocol/contracts/utils/OwnableCallForwarder.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+import "./interfaces/IOwnableCallForwarder.sol";
+
+/// @title Contract that forwards the calls that its owner sends
+/// @dev AccessControlRegistry users that want their access control tables
+/// to be transferrable (e.g., a DAO) will use this forwarder instead of
+/// interacting with it directly. There are cases where this transferrability
+/// is not desired, e.g., if the user is an Airnode and is immutably associated
+/// with a single address, in which case the manager will interact with
+/// AccessControlRegistry directly.
+contract OwnableCallForwarder is Ownable, IOwnableCallForwarder {
+    /// @notice Forwards the calldata to the target address if the sender is
+    /// the owner and returns the data
+    /// @dev This function emits its event after an untrusted low-level call,
+    /// meaning that the order of these events within the transaction should
+    /// not be taken seriously, yet the content will be sound.
+    /// @param forwardTarget Target address that the calldata will be forwarded
+    /// to
+    /// @param forwardedCalldata Calldata to be forwarded to the target address
+    /// @return returnedData Data returned by the forwarded call
+    function forwardCall(
+        address forwardTarget,
+        bytes calldata forwardedCalldata
+    ) external payable override onlyOwner returns (bytes memory returnedData) {
+        returnedData = Address.functionCallWithValue(
+            forwardTarget,
+            forwardedCalldata,
+            msg.value
```

Comment:
Will this revert if the target is not `payable` or if the function name does not exist and there is no fallback function?

Reply:
I extended the tests here https://github.com/api3dao/airnode/pull/748/commits/322e91b620da13fcd02b2667875c333a3c8e0855
If the target is payable and `msg.value` of the call made to `forwardCall()` is zero, the forward doesn't revert. If the target is payable and `msg.value` of the call made to `forwardCall()` is not zero, the forward does revert.
If the function doesn't exist and there is no fallback function, the forward does revert.